subprojects/logging/src/test/groovy/org/gradle/internal/logging/LoggingCommandLineConverterTest.groovy

```diff
@@ -64,25 +64,26 @@ class LoggingCommandLineConverterTest extends Specification {
     }
 
     def convertsWarnLevel() {
-         expectedConfig.logLevel = LogLevel.WARN
+        expectedConfig.logLevel = LogLevel.WARN
 
-         expect:
-         checkConversion(['-w'])
-         checkConversion(['--warn'])
-     }
+        expect:
+        checkConversion(['-w'])
+        checkConversion(['--warn'])
+    }
 
     def convertsConsole() {
         expectedConfig.consoleOutput = consoleOutput
+        expectedConfig.logLevel = logLevel
 
         expect:
-        checkConversion([arg])
```

Comment:
:o: The method `checkConversion` should probably have an override method here that takes `String...`. It would make all test cases in the classes more readable I think. Not necessarily proposing that it needs to be done right now.